src/bun.js/webcore/body.zig

```diff
@@ -376,6 +377,14 @@ pub const Body = struct {
                 .Blob => this.Blob.size,
                 .InternalBlob => @as(Blob.SizeType, @truncate(this.InternalBlob.sliceConst().len)),
                 .WTFStringImpl => @as(Blob.SizeType, @truncate(this.WTFStringImpl.utf8ByteLength())),
+                .Locked => {
+                    if (this.Locked.readable) |readable| {
```

Comment:
I suggest moving this into a helper fn incase we use it in more places

Reply:
done in https://github.com/oven-sh/bun/pull/4127/commits/76b4e7bfdb810d77e2e8e36213563b00d77edb6f